lib/grpc/client/adapters/mint/connection_process/connection_process.ex

```diff
@@ -81,12 +81,18 @@ defmodule GRPC.Client.Adapters.Mint.ConnectionProcess do
         {:ok, State.new(conn, opts[:parent])}
 
       {:error, reason} ->
-        Logger.error("unable to establish a connection. reason: #{inspect(reason)}")
+        Logger.error(
+          "unable to establish a connection to #{scheme}://#{host}:#{port}. Reason: #{inspect(reason)}"
```

Comment:
```suggestion
          "unable to establish a connection to #{scheme}://#{host}:#{port}. reason: #{inspect(reason)}"
```

Reply:
Just a small suggestion to keep a pattern. Either we start with uppercase or lowercase letters, but it's not good to mix them.

Reply:
modified the PR as suggested